internal/k8s/watch.go

```diff
@@ -30,9 +30,19 @@ type watcher interface {
 	Watch(options metav1.ListOptions) (watch.Interface, error)
 }
 
+type gvrWithNamespaced struct {
+	schema.GroupVersionResource
+	namespaced bool // if true, this resource can exist within a namespace; if false, cluster-wide
```

Comment:
if the comment goes above the struct member then certain tools/IDEs will pull that comment in when you hover over a reference to it.